modules/page/html/render/settings.js

```diff
@@ -30,29 +36,51 @@ exports.create = function (api) {
         h('section.prepend', prepend),
         h('section.content', [
           h('section', [
-            h('h2', 'Theme'),
-            h('select', {
-              style: {
-                'font-size': '120%'
-              },
-              value: currentTheme,
-              'ev-change': (ev) => api.settings.sync.set({
-                patchwork: {theme: ev.target.value}
+            h('h2', i18n('Theme')),
+            computed(currentTheme, currentTheme => {
```

Comment:
When you rebase/merge with the latest master, you'll see that the theme picker has been [replaced with a `select` dropdown box](https://github.com/ssbc/patchwork/commit/df3a0af70b34b408b510eafb21126103772656a3) instead of the toggle buttons. I suggest you use that for the language picker too!

Reply:
Oh wait, I see that this has already merged in the change. But it has overridden it with the old code putting the original theme picker back in.

Can you update this to use the new code from https://github.com/ssbc/patchwork/commit/df3a0af70b34b408b510eafb21126103772656a3 ?